content/docs/components-and-props.md

```diff
@@ -16,23 +16,24 @@ prev: rendering-elements.html
 next: state-and-lifecycle.html
 ---
 
-Components let you split the UI into independent, reusable pieces, and think about each piece in isolation. This page provides an introduction to the idea of components. You can find a [detailed component API reference here](/docs/react-component.html).
+componentहरूबाट तपाईंले UI लाई स्वतन्त्रा रूपमा  विभाजन  गर्न , पुन: प्रयोज्य टुक्राहरूमा विभाजन गर्न  र प्रत्येक टुक्रालाई अलग अलग  रूपमा सोच्न सक्नुहुन्छ।यस पृष्ठले componentsको विचारलाई  परिचय प्रदान गर्दछ। तपाईंले यो linkमा क्लिक गर्नुभयो भने  componet APIको बारेमा विस्तृत  जानकारी प्राप्त गर्न सक्नुहुन्छ [detailed component API reference here](/docs/react-component.html).
```

Comment:
Capitalize `Component` as its start of the sentence. 

I think there needs to be a space, between the Nepali additives (such as हरू and को) and English words, since them being different languages, what do you think?

There is a typo in `componet API`. Should be `Component API` 

Maybe add a Nepali Full stop at the end?

Typo in `स्वतन्त्रा` , Should be `स्वतन्त्र` 

There needs to be a space after full stop `।`